src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -49,6 +49,12 @@ class SubscriberArgs(parser: ArgParser) {
     )
 }
 
+enum class StatusCode(private val code: Int) {
+    SUCCESS(0), FAILURE(1);
```

Comment:
Not that I am feeling strong against this, but you can just declare enum constants and use their `ordinal` property: `enum class StatusCode { SUCCESS, FAILURE }`

Reply:
Thought, that we need to declare code explicitly.
Done